Exiled.CustomItems/Events/ServerHandler.cs

```diff
@@ -0,0 +1,58 @@
+// -----------------------------------------------------------------------
+// <copyright file="ServerHandler.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.CustomItems
+{
+    using System.Collections.Generic;
+    using System.Text;
+
+    using Exiled.API.Features;
+    using Exiled.CustomItems.API.Features;
+    using Exiled.Events.EventArgs;
+
+    using NorthwoodLib.Pools;
+
+    /// <summary>
+    /// Handles Server events for the CustomItems API.
+    /// </summary>
+    internal sealed class ServerHandler
+    {
+        /// <inheritdoc cref="Events.Handlers.Server.OnSendingRemoteAdminCommand"/>
+        public void OnRemoteAdminCommand(SendingRemoteAdminCommandEventArgs ev)
+        {
+            if (ev.Name.ToLower() == "gban-kick" && ev.Sender.ReferenceHub.queryProcessor._sender.ServerRoles.RaEverywhere)
+            {
+                Player player = Player.Get(string.Join(" ", ev.Arguments));
+                if (player == null)
+                    return;
+
+                if (CustomItem.TryGet(player, out IEnumerable<CustomItem> customItems))
+                {
+                    StringBuilder builder = StringBuilderPool.Shared.Rent();
+
+                    foreach (CustomItem item in customItems)
+                    {
+                        if (item.ShouldMessageOnGban)
+                        {
+                            builder.AppendLine(item.Name + " - " + item.Description);
```

Comment:
You should use `builder.Append(...).Append(...).AppendLine(...)` instead of contatenating strings.